src/app/app/(application)/dashboard/page.tsx

```diff
@@ -90,7 +93,7 @@ const PageDashboard = () => {
     <div className="relative overflow-hidden flex gap-3 p-5 max-w-screen-2xl mx-auto flex-col sm:flex-row">
       <div className="md:flex flex-col sm:w-[300px] w-full pb-5 mb-3 sm:mb-0 border-b border-gray-100 sm:border-none">
         <div className="flex flex-col w-full gap-5 rounded-lg bg-[#e7eaf133] px-2">
-          <div className="flex flex-col w-full h-24 gap-5 relative">
+          <div className="group/item flex flex-col w-full h-36 relative overflow-hidden">
```

Comment:
o que é esse group/item?

Reply:
Isso é uma forma para utilizar as pseudo classes. 

Por exemplo... 
Quando dizemos ao CSS que uma classe filha ao pai, precisa alterar o estado quando o PAI dela for hover

Reply:
```
.pai .child {
  visibility: hidden,
}

.pai:hover .child {
   visibility: visible
}
```